pkg/noobaaaccount/reconciler.go

```diff
@@ -373,6 +388,33 @@ func (r *Reconciler) CreateNooBaaAccount() error {
 		r.Logger.Errorf("got error on NooBaaAccount creation. error: %v", err)
 		return err
 	}
+	// create join secret conatining auth token for remote noobaa account
+	annotationValue, exists := util.GetAnnotationValue(r.NooBaa.GetAnnotations(), "remote-operator")
+	if exists {
+		if strings.ToLower(annotationValue) != strconv.FormatBool(false) {
+			res, err := r.NBClient.CreateAuthAPI(nb.CreateAuthParams{
+				System: r.Request.Name,
+				Role:   "operator",
+				Email:  options.OperatorAccountEmail,
+			})
+			if err != nil {
+				return fmt.Errorf("cannot create an auth token for remote operator, error: %v", err)
+			}
+			if res.Token == "" {
```

Comment:
Is an empty token somthing that can be returned by noobaa-core in case there is no error? did you encounter this?